wasm-node/src/wrapper/libp2p.rs

```diff
@@ -0,0 +1,72 @@
+use libp2p::swarm::{
+    ConnectionCounters as SwarmConnectionCounters, NetworkInfo as SwarmNetworkInfo,
+};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct NetworkInfo(SwarmNetworkInfo);
+
+impl From<SwarmNetworkInfo> for NetworkInfo {
+    fn from(info: SwarmNetworkInfo) -> NetworkInfo {
+        NetworkInfo(info)
+    }
+}
+
+#[wasm_bindgen]
+impl NetworkInfo {
+    #[wasm_bindgen(getter)]
+    pub fn num_peers(&self) -> usize {
+        self.0.num_peers()
+    }
+
+    pub fn connection_counters(&self) -> ConnectionCounters {
+        self.0.connection_counters().clone().into()
+    }
+}
+
+#[wasm_bindgen]
+pub struct ConnectionCounters(SwarmConnectionCounters);
+
+impl From<SwarmConnectionCounters> for ConnectionCounters {
+    fn from(info: SwarmConnectionCounters) -> ConnectionCounters {
+        ConnectionCounters(info)
+    }
+}
+
+#[wasm_bindgen]
+impl ConnectionCounters {
+    #[wasm_bindgen(getter)]
```

Comment:
do those getters support printing this object as a whole with eg `console.log`?

Reply:
yes 
<img width="332" alt="Screenshot 2023-10-27 at 17 42 22" src="https://github.com/eigerco/celestia-node-rs/assets/9157240/0d32412b-0b09-4cd7-85e6-1784d55d4bab">

the `(...)` is a side effect of this being only a getter not primitive value - one has to click on it for chrome to pull the actual value as I did for the first 3